public/app/core/components/AppChrome/OrganizationSwitcher/OrganizationSelect.tsx

```diff
@@ -10,12 +10,19 @@ import { OrganizationBaseProps } from './types';
 
 export function OrganizationSelect({ orgs, onSelectChange }: OrganizationBaseProps) {
   const styles = useStyles2(getStyles);
-  const { orgName: name, orgId, orgRole: role } = contextSrv.user;
-  const [value, setValue] = useState<SelectableValue<UserOrg>>(() => ({
-    label: name,
-    value: { role, orgId, name },
-    description: role,
-  }));
+  const { orgId} = contextSrv.user;
```

Comment:
```suggestion
  const { orgId } = contextSrv.user;
```
You might want to run some automatic formatting to avoid this in the future